### PR TITLE
new cs translation (histogram), typos, reformating

### PR DESCRIPTION
**Pull request by gamodulon**
translated histogram to cs, typos in histogram (en), reformated matlab_start headers (cs+en)

### CODE REVIEW

File: input/docs/en/histogram.md [hunk 1]
```diff
@@ -43,25 +43,27 @@ Enable user to pick a square and replacing it with equalized one
...
         imshow(z_ekv,'Parent',ax)
     end
     
-    [y,x, button]=ginput(1); %waiting for button press
+    [y,x, button]=ginput(1); % waiting for button press
 end
 ```
 ![](../media/zubr_anim.gif)
 ## Adaptive equalization
+Adaptive equalization computes separate histograms for parts of an image, which results in higher local contrast in the whole image.
```
Comment (tesar-tech) [inline input/docs/en/histogram.md]: 👍

File: input/docs/cs/histogram.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+figure;
+ax = axes; % uložení os do proměnné
+
+while button==1 % kliknutí levým tlačítkem na tlačítko
+    if x>100 && y >100 % vynechání okrajů snímku
+        z_ekv=C;
+        z_ekv(x-100:x+100,y-100:y+100)=histeq(z_ekv(x-100:x+100,y-100:y+100));
+        imshow(z_ekv,'Parent',ax)
+    end
+
```
Comment (tesar-tech) [inline input/docs/cs/histogram.md]: řádek netřeba

File: input/docs/cs/histogram.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+while button==1 % kliknutí levým tlačítkem na tlačítko
+    if x>100 && y >100 % vynechání okrajů snímku
+        z_ekv=C;
+        z_ekv(x-100:x+100,y-100:y+100)=histeq(z_ekv(x-100:x+100,y-100:y+100));
+        imshow(z_ekv,'Parent',ax)
+    end
+    
+    [y,x, button]=ginput(1); % čekání na zmáčknutí tlačítka
+end
+```
```
Comment (tesar-tech) [inline input/docs/cs/histogram.md]: tu chybí ten gif

Comment (gamodulon) [reply input/docs/cs/histogram.md]: c5b263b

**Pull request merged**